### PR TITLE
GA release for AzureCommunicationCommon 1.2.0

### PR DESCRIPTION
**Pull request by maximrytych-ms**
Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`. It will impact any code that previously depended on the use of `UnknownIdentifier` with rawIDs starting with `28:orgid:`, `28:dod:`, or `28:gcch:`.

### CODE REVIEW

**Review by AikoBB — APPROVED** [review]
Let's merge after getting approval of @tjprescott

**Review by mjafferi-msft — APPROVED** [review]

**Pull request merged**
